ui/resolution_selection.js

```diff
@@ -198,14 +208,26 @@ shaka.ui.ResolutionSelection = class extends shaka.ui.SettingsMenu {
   updateLocalizedStrings_() {
     const LocIds = shaka.ui.Locales.Ids;
 
-    this.button.setAttribute(shaka.ui.Constants.ARIA_LABEL,
-        this.localization.resolve(LocIds.RESOLUTION));
-    this.backButton.setAttribute(shaka.ui.Constants.ARIA_LABEL,
-        this.localization.resolve(LocIds.RESOLUTION));
-    this.backSpan.textContent =
+    // Assign text attributes depending on content type.
+    if (this.player.isAudioOnly()) {
+      this.button.setAttribute(shaka.ui.Constants.ARIA_LABEL,
```

Comment:
Looks like you could set a local variable with the localization ID, instead of duplicating all the setAttribute and resolve calls in each branch.

Reply:
Of course, that works way better - Updated it on my last commit! 

```js
// Assign text attributes depending on content type.
const locId = this.player.isAudioOnly() ? LocIds.QUALITY : LocIds.RESOLUTION;
```